packages/dart_frog/lib/src/request.dart

```diff
@@ -95,6 +95,12 @@ class Request {
 
   final shelf.Request _request;
 
+  /// Connection information for the associated HTTP request.
+  HttpConnectionInfo get connectionInfo {
+    // ignore: cast_nullable_to_non_nullable
+    return _request.context['shelf.io.connection_info'] as HttpConnectionInfo;
```

Comment:
```suggestion
    return _request.context['shelf.io.connection_info']! as HttpConnectionInfo;
```

Should prevent an ignore right?

Reply:
Good call will update shortly